lib/triplex.ex

```diff
@@ -172,9 +170,13 @@ defmodule Triplex do
     end
   end
 
-  defp error_message(%PGError{} = e), do: PGError.message(e)
-  defp error_message(%MXError{} = e), do: MXError.message(e)
-  defp error_message(msg), do: msg
+  defp error_message(msg) do
+    if Exception.exception?(msg) do
```

Comment:
That looks perfect! Unfortunately they don't have a defguard for it on elixir, it would be a lot more idiomatic if you could use `when Exception.exception?(msg)` on a guard, but it's not possible as it is now. I'll open a PR to Elixir core and check if they accept it though.